doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -243,7 +243,7 @@ terminating NUL byte.
 
 EVP_PKEY_CTX_set_rsa_padding() sets the RSA padding mode for I<ctx>.
 The I<pad> parameter can take the value B<RSA_PKCS1_PADDING> for PKCS#1
-padding, B<RSA_SSLV23_PADDING> for SSLv23 padding, B<RSA_NO_PADDING> for
```

Comment:
Please keep it here, just add deprecated. `deprecated B<RSA_SSLV23_PADDING> for SSLv23 padding`

Reply:
I put it as `(deprecated)` to follow the structure of the other options.   Whew, that's a long sentence, should probably be turned into a bullet-list but that's for another PR.